src/utils/NodeHelper.ts

```diff
@@ -45,13 +45,12 @@ export class NodeHelper {
     return keypath.replace(/\[(.*?)\]/g, '.$1').split('.')
   }
 
-  static getPathWithoutNamespace(keypath: string, node?: Node, namespace?: string) {
+  static getPathWithoutNamespace(keypath: string, node?: Node, namespace?: string, delimiter = '.') {
     if (Global.namespaceEnabled) {
       namespace = node?.meta?.namespace || namespace
-      if (namespace)
+      if (namespace && keypath.startsWith(namespace + delimiter))
         return keypath.slice(namespace.length + 1)
```

Comment:
[Not this PR's fault]

Shouldn't this do the slice based on the length of the delimiter (for robustness). Can we assume that a delimiter will always be a single character?